src/lib/AutomateModule.ts

```diff
@@ -2,21 +2,23 @@
 /* eslint-disable no-empty */
 import { BigNumber, ethers } from "ethers";
 import {
-  Web3FunctionParams,
-  Web3FunctionUserArgs,
-  Web3FunctionUserArgsSchema,
   Module,
   ModuleArgsParams,
   ModuleData,
   ResolverParams,
   TimeParams,
+  TriggerParams,
+  Web3FunctionParams,
   Web3FunctionSchema,
+  Web3FunctionUserArgs,
+  Web3FunctionUserArgsSchema,
 } from "../types";
+import { TriggerConfig, TriggerType } from "../types/Trigger.interface";
```

Comment:
Should import directly from `../types`

Reply:
fixed d99d312655f980f580bb981a4c607d35f34eec09